### PR TITLE
adding weather api lab

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Will-max-bit — COMMENTED** [review]
looks like there is an error when the quotes lab is run.

**Comment by ghost** [comment]
Which one? 18 or 18b?

File: code/scott/python/lab_18b_quotes.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import requests
+more_quotes = 'yes'
+next_page = 'yes'
+page = 0
+while more_quotes == 'yes':  
+    keyword = input('\nEnter keyword to seach for quotes: ')
+    while next_page == 'yes':
+        page += 1
+        url = (f'https://favqs.com/api/quotes?page={page}&filter={keyword}')
+        headers = {'Authorization': 'Bearer xxxxxxxxxxxxxxxxxxxxxxxxxxx'}
```
Comment (AlexDeMichieli) [inline code/scott/python/lab_18b_quotes.py:10]: That should be headers = {'Authorization': 'Token token="xxxxxxxx"'}

File: code/scott/python/oop/bank.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from types import ClassMethodDescriptorType
```
Comment (AlexDeMichieli) [inline code/scott/python/oop/bank.py:1]: I don't think we this this here

Comment (ghost) [reply code/scott/python/oop/bank.py:1]: VS Code added that automatically so I left it in.

**Review by AlexDeMichieli — APPROVED** [review]

**Pull request merged**